<?xml version='1.0' encoding='utf-8'?>
<ui version="4.0">
  <!-- Generated for PYSIDE6 -->
  <class>TabOrderTest</class>
  <widget class="QDialog" name="TabOrderTest">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>300</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>TabOrderTest</string>
    </property>
    <widget class="QLineEdit" name="field1">
      <property name="geometry">
        <rect>
          <x>50</x>
          <y>50</y>
          <width>200</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLineEdit" name="field2">
      <property name="geometry">
        <rect>
          <x>50</x>
          <y>90</y>
          <width>200</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QPushButton" name="submitBtn">
      <property name="text">
        <string>Submit</string>
      </property>
      <property name="geometry">
        <rect>
          <x>50</x>
          <y>130</y>
          <width>100</width>
          <height>30</height>
        </rect>
      </property>
    </widget>
  </widget>
  <tabstops>
    <tabstop>field1</tabstop>
    <tabstop>field2</tabstop>
    <tabstop>submitBtn</tabstop>
  </tabstops>
</ui>
